app/assets/javascripts/pageflow/chart/page_type.js

```diff
@@ -18,10 +18,14 @@ pageflow.pageType.register('chart', _.extend({
 
   resize: function(pageElement, configuration) {
     var iframeWrapper = pageElement.find('.iframeWrapper'),
-      pageHeader = pageElement.find('.page_header'),
-      scroller = pageElement.find('.scroller');
+        pageHeader = pageElement.find('.page_header'),
+        scroller = pageElement.find('.scroller'),
+        fullWidth = configuration.full_width,
+        widescreened = pageElement.width() > 1430;
 
-    var widescreened = pageElement.width() > 1430;
+    if(fullWidth) {
```

Comment:
Even if this particular file already is quite messy, the general coding style in Pageflow is to use a blank after `if`. Some of the other repos already have a `.jshintrc`. We should probably add add this for this repository, too, and activate Hound to ensure each PR moves things into the right direction.